### PR TITLE
 add MYSQL question from line number 380 to 390

### PR DESCRIPTION
**Pull request by Moksh45**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -377,6 +377,19 @@ const sqlQuiz = [
...
+  {
+    Question:
+      "In SQL If You Want To Undo a GRANT, you Should use",
+    Answer: "REVOKE",
+    Distractor1: "UNDO",
+    Distractor2: "DELETE",
+    Distractor3: "ROLLBACK",
+    Explanation:
+      "The REVOKE statement can be used to undo the action of a GRANT statement.",
+    Link: "https://dev.mysql.com/doc/refman/5.6/en/revoke.html#:~:text=The%20REVOKE%20statement%20enables%20system,1.4%2C%20%E2%80%9CGRANT%20Statement%E2%80%9D."
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:389]:
```suggestion
    Link: "https://dev.mysql.com/doc/refman/5.6/en/revoke.html"
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -377,6 +377,19 @@ const sqlQuiz = [
       "You can select from INFORMATION_SCHEMA.COLUMNS and combine with a WHERE clause when needing to identify where a certain column is stored.",
     Link: "https://www.freecodecamp.org/news/learn-sql-queries-database-query-tutorial-for-beginners/",
   },
+  {
+    Question:
+      "In SQL If You Want To Undo a GRANT, you Should use",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:382]:
```suggestion
      "In SQL, what statement should you use to undo a GRANT statement?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 2 above)
Comment (Moksh45) [reply src/data/sql-quiz.ts:382]: Ok, I am trying to solve this issue

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**